crates/ev-cli/src/commands/enclave/init.rs

```diff
@@ -9,15 +8,15 @@ use ev_enclave::config::{
 /// Initialize an Enclave.toml in the current directory
 #[derive(Debug, Parser)]
 #[command(name = "init", about)]
-#[command(group(
+#[clap(group(
```

Comment:
this was a breaking change in clap 4 that only appears at runtime (??).. Really need to do a good bug bash before we release this because of the amount of stuff that's changing in the overall CLI merge